Header header
float32 compass
float32 heading
float32 heading_rate
float32 gyro_bias
float32 stddev
